modin/engines/base/io/io.py

```diff
@@ -266,7 +266,7 @@ def _read(cls, **kwargs):
         return pd_obj
 
     @classmethod
-    @_inherit_docstrings(pandas.read_json, apilink="pandas.read_json")
+    @_inherit_docstrings(pandas.read_json, apilink="pandas.io.json.read_json")
```

Comment:
Is this the only change to fix the issue?

Reply:
Yes, according to the [failed test](https://github.com/modin-project/modin/runs/3069668334#step:6:35) - this is the only broken link.